### PR TITLE
Localize claim page

### LINKED ISSUE #75: Localize Claim page

### PR DESCRIPTION
**Pull request by alecrem**
Closes #75
Was: #76

- Localize claim page
- Localize Claim link on nav bar
- Bonus: localize commented out Badges link
- Bonus fix: dash length had become different between languages

### CODE REVIEW

**Review by geeknees — APPROVED** [review]
LGTM!

`受け取り`seems to be better using Kanji characters. I aslo thought it would be better to keep the original expression for `Claim` in the menu, so I changed it to katakana.

File: locales/ja/claim.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "heading": {
+    "title": "$HENKAKUを受けとる",
+    "getKamonFirst": "まずはKamon NFTを取得しましょう！",
+    "claim": "トークンを受けとろう！"
+  },
+  "info": {
+    "canClaim": "$HENKAKU受けとり可能です。",
+    "cannotClaim": "$HENKAKUを受けとるにはKamon NFTが必要です。",
+    "hasThisManyPoints": "現在{{points}} $HENKAKUを受けとることができます。",
+    "hasNoPoints": "しかし、$HENKAKUを受けとれるほどのポイントをお持ちではありません。"
```
Comment (geeknees) [inline locales/ja/claim.json:11]:
```suggestion
    "hasNoPoints": "しかし、交換可能な$HENKAKUを持っていません。"
```

**Comment by alecrem** [comment]
@geeknees Thank you for taking a look!

> I aslo thought it would be better to keep the original expression for `Claim` in the menu, so I changed it to katakana.

Would クレーム look like "complaints"? Note that this will be on the nav bar, completely out of context when shown on other pages.

Any other way to say it in Japanese might be too long for the nav bar. Should we consider keeping "Claim" in English? And Changing "Quests" to alphabet, too.

A nav bar being completely in English is not rare in Japanese websites, and I think in our case it can make sense.

**Comment by geeknees** [comment]
> A nav bar being completely in English is not rare in Japanese websites, and I think in our case it can make sense.

This choice may be a good one. Since this app also serves as a gate for web3, I think it is better to use English in the sense of learning basic web3 terminology.

**Comment by alecrem** [comment]
@geeknees Let's just merge this with "Claim", which also leaves the UI unchanged wrt this link, and follow the conversation on Discord or make a different issue.

**Review by geeknees — APPROVED** [review]

**Pull request merged**